### PR TITLE
Add beta status

### PR DESCRIPTION
**Pull request by CPatchane**
* [x] Drawer
* [x] Settings menu
* [x] Next to the app name

### CODE REVIEW

**Comment by CPatchane** [comment]
[(3) ⭐️ [DIV-2] ETU j'ai conscience d'être sur une version beta](https://trello.com/c/YXTAPdtU/361-3-div-2-etu-j-ai-conscience-d-etre-sur-une-version-beta)

File: src/styles/navigation_section.css [hunk 1]
```diff
@@ -90,6 +96,10 @@
     padding: 0 .5em;
 }
 
+[role=banner] .coz-nav-group:not(.coz-nav--error).coz-nav-group--grey {
```
Comment (gregorylegarec) [inline src/styles/navigation_section.css]: Is `--grey` the  best semantic specifier ? Maybe something like `--inactive` should be better ? What do you think ?

Comment (CPatchane) [inline src/styles/navigation_section.css]: Indeed, that sounds better for the semantic here, I agree 👍

**Comment by gregorylegarec** [comment]
Great @CPatchane !

**Pull request merged**